src/Games/NexusMods.Games.RedEngine/Cyberpunk2077/Emitters/ArchiveXLMissingEmitter.cs

```diff
@@ -0,0 +1,26 @@
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Games.DTO;
+using NexusMods.Abstractions.NexusWebApi.Types;
+using NexusMods.Paths;
+
+namespace NexusMods.Games.RedEngine.Cyberpunk2077.Emitters;
+
+/// <summary>
+/// Diagnostic emitter for missing ArchiveXL when a mod installs .xl files in the game folder.
+/// </summary>
+public class ArchiveXLMissingEmitter : APathBasedDependencyEmitterWithNexusDownload
+{
+    protected override string DependencyName => "ArchiveXL";
+
+    protected internal override IEnumerable<GamePath> DependencyPaths => new[]
+    {
+        // ArchiveXL DLL
+        new GamePath(LocationId.Game, "red4ext/plugins/ArchiveXL/ArchiveXL.dll"),
+    };
+
+    protected internal override GamePath[] DependantPaths => [new GamePath(LocationId.Game, "")];
```

Comment:
This seems odd and can probably just be this:

```csharp
DependantPaths { get; } = [];
```

Reply:
This is a bit different it says "any file in the game path", the version you suggested would be no paths at all. This version looks strange but it's  `{Game}/`